op-guide/build.md

```diff
@@ -0,0 +1,145 @@
+# Build
+
+## Supported platforms
+
+The following table lists TiDB support for common architectures and operating systems. 
+
+|Architecture|Operating System|Status|
+|------------|----------------|------|
+|amd64|Linux Ubuntu (14.04+)|Stable|
+|amd64|Linux CentOS (7+)|Stable|
+|amd64|Mac OSX|Experimental|
+
+## Prerequisites
+
++ Go [1.5+](https://golang.org/doc/install)
++ Rust [nightly version](https://www.rust-lang.org/downloads.html)
++ GCC 4.8+ with static library
+
+### Install GO
+
+#### Linux
```

Comment:
Is it really necessary to provide so many details on how to install Go, Rust and GCC on different platforms in our doc? If you think it's really necessary, please:
1. Add links after each prerequisite to the corresponding section.
2. Add unordered list to each of the platform.